### DISCUSSION TITLE
Randomly stuck on beatmap selector (Screenshots are not what I saw)

### DISCUSSION THREAD

**Discussion by rvnexe** [ppy/osu#22781] [Q&A]
### Type

Game behaviour

### Bug description

(I've experienced this bug so much before, but didn't posted it)

This time:  I was reading the wiki. then closed the wiki and opened music player. I realized that the three line button is not woking.
closed it and dragged the beatmap list, but all beatmaps was blank (without any icon, photo or text) and while dragging, I saw a bright one in the list, that I think was the highlighted beatmap.
But information and ranking of that highlighted beatmap was on the left of the screen and woking correctly. also, background too.
I got 2 screenshots of this situation, but the photos are just a loading screen! 

I'll take photo with camera next time and post here.

### Screenshots or videos

![Screenshot_43](https://user-images.githubusercontent.com/89677246/222746829-4ba3253e-cf9a-4f5c-a040-3d4e91dd08ec.png)
![Screenshot_44](https://user-images.githubusercontent.com/89677246/222746834-f07563cf-9fbc-41aa-aff0-ddb16402f094.png)

### Version

2023.301.0 lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/10882905/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10882906/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10882907/updater.log)
[database.log](https://github.com/ppy/osu/files/10882908/database.log)
[input.log](https://github.com/ppy/osu/files/10882909/input.log)
[network.log](https://github.com/ppy/osu/files/10882910/network.log)

**Comment by rvnexe** [comment]
Update:  F12 wasn't woking as a screenshot bind

**Comment by bdach** [comment]
I've downgraded this to a discussion for now because I can't make heads or tails of the opening post. You're saying you're stuck on "beatmap selector", but screenshots are showing the gameplay load screen?

Please capture this with an external camera the next time this happens because I'm not sure what is going on here.

**Reply by rvnexe** [reply]
ok, i'll
